big_tests/tests/mam_SUITE.erl

```diff
@@ -3074,6 +3084,24 @@ metric_incremented_on_archive_request(ConfigIn) ->
     Config = [{mongoose_metrics, MongooseMetrics} | ConfigIn],
     escalus_fresh:story(Config, [{alice, 1}], F).
 
+metrics_incremented_for_async_pools(Config) ->
+    Val0 = get_mongoose_async_metrics(),
+    archived(Config),
+    Val1 = get_mongoose_async_metrics(),
+    ?assert_equal(false, Val0 =:= Val1).
+
+get_mongoose_async_metrics() ->
+    HostType = domain_helper:host_type(mim),
+    HostTypePrefix = domain_helper:make_metrics_prefix(HostType),
+    #{batch_flushes => get_mongoose_async_metrics(HostTypePrefix, batch_flushes),
+      timed_flushes => get_mongoose_async_metrics(HostTypePrefix, timed_flushes)}.
+
+get_mongoose_async_metrics(HostTypePrefix, MetricName) ->
+    Metric = [HostTypePrefix, mongoose_async_pools, pm_mam, MetricName],
+    {ok, Value} = rpc(mim(), exometer, get_value, [Metric]),
```

Comment:
use `mongoose_metrics:get_metric_value/1`. Just in case we move away from exometer :)

Reply:
Done 👍🏽